bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -127,6 +129,16 @@
 		<category>Energy</category>
 		<state pattern="%.0f" readOnly="true" />
 	</channel-type>
+	<channel-type id="water_box_status">
+		<item-type>Switch</item-type>
```

Comment:
The type here is switch, but you send a decimal. Should the type here not be matched to it?
What is the mihome map showing? A switch or a number?

Reply:
The type should already be a switch, because the status flag is a bool.

I oriented myself to the dnd_enabled switch:
https://github.com/openhab/openhab-addons/blob/e809c838d229e42f8262fbc6e732b1ef06aebdae/bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml#L107-L110
https://github.com/openhab/openhab-addons/blob/e809c838d229e42f8262fbc6e732b1ef06aebdae/bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/MiIoBindingConstants.java#L49

There it was solved the same way:
https://github.com/openhab/openhab-addons/blob/e809c838d229e42f8262fbc6e732b1ef06aebdae/bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoVacuumHandler.java#L134